style/styles.css

```diff
@@ -0,0 +1,1101 @@
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  margin-top: 80px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.personOfWeek div {
+  font-family: "Cormorant Garamond", serif;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  margin: 20px;
+}
+
+.personOfWeekProfileImageDiv {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  width: 260px;
+  border: 5px solid #d33b2b;
+  border-radius: 11px;
+  vertical-align: middle;
+  box-shadow: 0 20px 40px 10px rgb(54, 52, 52, 0.8);
+  transform: rotate(-15deg);
+  transform-origin: left bottom;
+}
+
+.personOfWeekProfileImageDiv:hover {
+  transform: rotate(0);
+  box-shadow: 0 20px 500px 10px rgb(54, 52, 52, 0);
+}
+
+.personOfWeekText {
+  width: 1100px;
+}
+
+.personOfWeek h2 {
+  font-size: 25px;
+  margin-bottom: 30px;
+}
+
+.linkTexts {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+}
+
+.linkTexts h2 {
+  color: #666;
+  margin-bottom: 20px;
+}
+
+.linkTexts h1 a:hover {
+  color: #d33b2b;
+}
+
+.shippingNow #element5 a {
+  background: #fff;
+  color: #277dcc;
+  padding: 30px;
+  margin-right: 25px;
+  margin-left: 25px;
+  font-size: 20px;
+  border-radius: 12px;
+}
+
+.communityLinks {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  color: black;
+  margin-top: 70px;
+}
+
+.linkContainer {
+  display: flex;
+  justify-content: start;
+  flex-direction: row;
+  margin-left: auto;
+  margin-right: auto;
+  width: 700px;
+  height: 300px;
+}
+
+@media (max-width: 800px) {
+  .linkContainer {
+    display: flex;
+    justify-content: start;
+    flex-direction: row;
+    flex-wrap: wrap;
+    margin-left: auto;
+    margin-right: auto;
+    width: auto;
+    height: auto;
+  }
+
+  .communityLinks {
+    display: grid;
+    grid-template-columns: 1fr;
+    color: black;
+    margin-top: 70px;
+  }
+}
+
+.linkContainer .imgContainer {
+  margin: auto;
+  margin-top: 0;
+  padding-right: 30px;
+}
+
+.linkContainer .imgContainer .profileImg {
+  border: 5px solid #d33b2b;
+}
+
+.dateAndComment {
+  display: flex;
+  flex-direction: row;
+}
+
+.wholeCommunityLinks {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+
+.more-content-button {
+  font-family: "Cormorant Garamond", serif;
+  color: #333;
+  text-decoration: none;
+  border-bottom: #d33b2c 0.225em solid;
+}
+
+.seeAllLatestArticles {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 100px;
+}
+
+.more-content-button:hover {
+  font-family: "Cormorant Garamond", serif;
+  color: #d33b2c;
+}
+
+.wholeFeaturedBook {
+  margin-top: 60px;
+  background-color: #d33b2b;
+  color: white;
+}
+
+.featuredBook {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap-reverse;
+  height: 400px;
+  color: white;
+  width: 86%;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.featuredBookText {
+  margin-top: 400px;
+  flex: 70%;
```

Comment:
This could be a potential cause for the horizontal scrollbar in your clone.

Reply:
It is all done.